edubots_surveys.Rnw

```diff
@@ -1417,45 +1178,12 @@ teacher, with the use of messaging apps. The findings are summarized in Table
 teachers do not want to participate in a chat group with students; either they
 want to simply leave the students alone in their own chat group, or otherwise
-they prefer not to be part of that interaction. In general, that is going to be
-the case no matter what; it is well known that students organize their own chat
-groups with many (and not always conveyable) intentions, so teachers do not want
-to take any part in these informal or non institutionally-supported chat
-groups. Overwhelmingly, they do not want to participate in these kind of chat
-groups with students, but even less so if it includes the rest of the faculty.
-
-The answer, then, to this research question, is that, in general, teachers do
-{\em not} want to have interaction with students in a chat group. To a certain
-point, this would seem to contradict results by Gachago et
-al. \cite{gachago2015crossing}, although this might be due to cultural
-attitudes, or other factors such as the average size of classes. It does
-confirm, however, that the challenges cited in that study, and possibly others,
-are an obstacle to the adoption of mobile (and other) instant messaging among
-the community of surveyed teachers.
-
-Regarding the social factor of chat groups that teachers use in class, according
-to Fig. \ref{figure:chatbots_messagingorganization} the vast majority prefer
-small groups only with the students from the same course. These are more focused
-groups with specific goals and dedicated to the organization of the course and
-its tasks, and from the pedagogic point of view seem more appropriate to improve
-the learning process. Interestingly, and as a cross-check of the answers above,
-about 30\% of teachers consider they should not be part of the chat group. This
-might seem to contradict the results of the other survey, but in fact, being a
-result of different surveys, to a certain point affirms the same thing: there is
-a great amount of teachers that would be against being in a chat group with
-students. Whether that's a slim majority or not, that's open to debate (and
-would probably need a more pointed survey focused on this), but what is clear is
-that forcing teachers to create chat groups with students and participate in
-them would create a certain amount of resistance. Also, only university teachers
-find interesting a group with all the students and teachers in their own Faculty
-or School. The lack of teachers from vocational education here may be the
-consequence of using specific language such as ``Faculty''. Moreover, the fact
-that many universities and schools already use these groups for administrative
-and social interaction (e.g. \cite{dibitonto2018chatbot}) might be the reason
-for the low percentage of teachers that selected this response.
+they prefer not to be part of that interaction. 
+
+%what followed, should be part of the discussion 
```

Comment:
I would have preferred to have atomic changes, again. That way we can decide, change by change, if it's OK or not (and also add the change marker to it.